Add a first year announcement (#244)
fortran-lang/fortran-lang.org: _posts/2021-04-20-First-Year.md

---
layout: post
title: "First year of the Fortran website"

> category: newsletter

Author: We have to change the category...

Reviewer: Can it be just "post"?

Reviewer 2: The 'category' isn't displayed anywhere on site and it looks like the site is currently hard-coded to only display posts in the 'newsletter' category, which needs fixing.

https://github.com/fortran-lang/fortran-lang.org/blob/a80ff0e335b18500e775cd0fd62cbbded78a9dfd/news/index.html#L12

Author: I see. I changed it to "post", but that would not show up, so I changed it back to "newsletter".

author: Ondřej Čertík, ...
---

In April 2020 we created a website for the Fortran language at

> [fortran-lang.org](https://fortran-lang.org/). In exactly one year, it has
> gotten to be the first result when you search "Fortran" in Bing, Yahoo,

Reviewer: ```suggestion
[fortran-lang.org](https://fortran-lang.org/). In exactly one year, it grew
￼to be the first result when you search "Fortran" in Bing, Yahoo,
```

DuckDuckGo, Ecosia, Qwant, SearchEncrypt and the second result in Google (after
the Wikipedia page for Fortran).

> The goal of the Fortran website is to maintain a neutral website where any
> Fortran user (expert or novice), compiler vendor (open source or commercial),
> Fortran Standards Committee member, enthusiast, supporter or anybody else
> interested is welcome to participate. The Fortran language has been invented in
> 1956, and we are trying to be the stewards of the language and we welcome you
> to join us.

Reviewer: ```suggestion
The goal of the website is to maintain a neutral place where any
￼Fortran user (expert or novice), compiler vendor (open source or commercial),
￼Fortran Standards Committee member, enthusiast, supporter or anybody else
￼interested is welcome to participate. Fortran was invented in
￼1956, and we aim to be the stewards of the language and we welcome you
￼to join us.
```
